tests/test_clan_members_war.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from mockito import when
+
+from tests.resources import clash_royale_client_riverracelog, clash_royale_client_responses
+from clash_royale_service import ClashRoyaleService
+from common.schemas import player_historical_activity
+
+# Test Data here
+mock_clan_players_war_history = {
+    '#YV9GU2VG': player_historical_activity(tag='#YV9GU2VG', name='Goku', role='member', exp_level=13, fame_hist=[None, None, None, None], boat_attacks_hist=[None, None, None, None], deck_used_hist=[None, None, None, None]), 
+    '#8VUG0GQRY': player_historical_activity(tag='#8VUG0GQRY', name='joseph', role='elder', exp_level=13, fame_hist=[None, None, None, None], boat_attacks_hist=[None, None, None, None], deck_used_hist=[None, None, None, None]), 
+    '#LYJVYUUUR': player_historical_activity(tag='#LYJVYUUUR', name='ZEPOL 1244', role='coLeader', exp_level=13, fame_hist=[None, None, None, None], boat_attacks_hist=[None, None, None, None], deck_used_hist=[None, None, None, None]),
+    '#SOMENOBDY': player_historical_activity(tag='#SOMENOBDY', name='NOBODY', role='member', exp_level=10, fame_hist=[None, None, None, None], boat_attacks_hist=[None, None, None, None], deck_used_hist=[None, None, None, None])
+}
+
+class TestRiverRaceLog(unittest.TestCase):
+    def test_past_weeks_clan_war(self):
+        clan_members_war = ClashRoyaleService()
+        clan_tag = "#9GULPJ9L"
+        # Mock API responses and certain function calls
+        when(clan_members_war.clash_royale_client).get_river_race_log(clan_tag).thenReturn(
+            clash_royale_client_riverracelog.RIVER_RACE_LOG_API_RESPONSE
+        )
+
+        clan_players_war_history = clan_members_war.past_weeks_clan_war(mock_clan_players_war_history, clan_tag)
+        # Testing for the length returned. Should be 3 players returned
+        self.assertEqual(len(clan_players_war_history), 4)
+        pass

```

Comment:
lmao, your test never reaches the below asserts since you have this `pass` here. Remove unnecessary pass